src/pages/Search/AdvancedSearchFilters.tsx

```diff
@@ -209,6 +209,12 @@ function AdvancedSearchFilters() {
                 description: 'common.to' as const,
                 route: ROUTES.SEARCH_ADVANCED_FILTERS_TO,
             },
+            {
+                title: getFilterDisplayTitle(searchAdvancedFilters, CONST.SEARCH.SYNTAX_FILTER_KEYS.HAS, translate),
+                description: 'search.filters.has' as const,
+                route: ROUTES.SEARCH_ADVANCED_FILTERS_HAS,
+                shouldHide: searchAdvancedFilters?.type !== CONST.SEARCH.DATA_TYPES.CHAT,
```

Comment:
Can we use `shouldShow` instead and add this field to all of them? We should return false for other expense only filters, e.g. `amount`, `currency`, `merchant`, `total`, `category`, `tag`, `tax rate`, `expense type` so we only show relevant filters based on the type you're currently seeing